tasks/managed/create-pyxis-image/README.md

```diff
@@ -27,6 +27,9 @@ The relative path of the pyxis.json file in the data workspace is output as a ta
 | taskGitUrl              | The url to the git repo where the release-service-catalog tasks and stepactions to be used are stored                                                                                                                                                                                                                                                                                                       | No       | ""                      |
 | taskGitRevision         | The revision in the taskGitUrl repo to be used                                                                                                                                                                                                                                                                                                                                                              | No       | ""                      |
 
+## Changes in 5.0.0
+* The default value of `.pyxis.skipLayers` now defaults to true instead of false
```

Comment:
Hmm, would it make more sense to rename it to the opposite then? Like `includeLayers` or `calculateLayers` or `enableLayers`? Since now it can be considered something optional/extra that you need to enable explicitly.

BTW, a side note, in case you missed it, the layer id is no longer required to be unique in Pyxis, so we shouldn't worry about that. But performance is a valid concern.

Reply:
> BTW, a side note, in case you missed it, the layer id is no longer required to be unique in Pyxis, so we shouldn't worry about that.

Thanks! I _did_ miss this.

Reply:
> would it make sense to rename it to the opposite then? Like includeLayers or calculateLayers or enableLayers?

I'm fine with this either way. Among the options you proposed, I think I would choose `includeLayers`.

Reply:
OK, let's go with `includeLayers` then. Given that it shouldn't break anything anymore, it should be ok to just replace one with the other without keeping both for a while. And in the konflux-release-data repo, you could also add includeLayers=true for the ubi images (which I think is the only one where we want it, right?). that way, once this change makes it to production, things continue to work the same way. and then at some point we'll just remove skipLayers from all RPAs. (I can create a Jira for that so that it's not forgotten.) WDYT?

If you'd prefer just your original change for now and do all of this later, that's fine with me.

Reply:
(I'm behind on all the things and haven't had a chance to get back to this yet. Busy with new-cluster automation this week.)

Reply:
Sorry for the delay. I pushed the proposed change to use includeLayers in a new commit. Please check if that looks ok to you.

Reply:
Sounds good to me. Thanks @mmalina !